### ISSUE TITLE
How to get the closed rate of the day before ?

### ISSUE THREAD

**Issue by traitrani** [madisvain/exchangeratesapi#125]
Hello,

I am using the paid membership and getting the rate with the date of yesterday, am I receiving the rate of the day before as defined on the European Central Bank ?

For example, if I want to get the EUR-RON rate of yesterday, a get / 2022-02-22 will give me the answer **4,9464** ?

I don't want to pay if, at the end, I don't have the correct use of the API.

Thanks in advance.

Kind regards,

Julien H.

**Comment by lucasRolff** [comment]
Yes, you'll receive the rate for yesterday when calling `/2022-02-22`. You can easily verify it using the free tier as well.

Keep in mind, there's multiple free services available still, even after exchangeratesapi.io got acquired.

E.g. my own version would give you the following response: https://exchangeapi.io/api/2022-02-22 (The service is actively in use by quite a few companies, and it's free to run, so won't require anything, not even an API key :P)
